ci/unit/test_post_always.sh

```diff
@@ -13,17 +13,32 @@ else
   exit 1
 fi
 
-rm -rf run_test.sh vm_test
 DAOS_BASE="${SL_PREFIX%/install*}"
 NODE="${NODELIST%%,*}"
-
 mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
 
-# shellcheck disable=SC2029
-ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
-  "DAOS_BASE=$DAOS_BASE             \
-  $(cat "$mydir/test_post_always_node.sh")"
+if [ $# -ge 1 ] && [ -n "$1" ]; then
+    if [ "$1" == "memcheck" ]; then
+        echo "run test post always with memcheck"
+        rm -rf run_test_memcheck.sh
```

Comment:
I don't think you really need to have all of this conditional code around creating different named results files/dirs.  This stage runs in a completely different namespace than the non-memcheck unit test so there won't be any file/dir name collisions until artifact collection time.  It would reduce much of this code change if you kept the same filenames in the functions and just did one final directory rename prior to artifact collection.

Reply:
okay, I'll simplify this part and do the rename on `test_post_always_node.sh` section, thanks